docs/examples.rst

```diff
@@ -547,4 +547,88 @@ Example of running this configurations are:
     # Set the mode to PRELOAD
     export DLIO_PROFILER_INIT=PRELOAD
     # Enable profiler
-    DLIO_PROFILER_ENABLE=1
\ No newline at end of file
+    DLIO_PROFILER_ENABLE=1
+
+
+Resnet50 with pytorch and torchvision example from ALCF Polaris:
+****************************************************************** 
+
+Create a separate conda environment for the application and install dlio-profiler
+
+.. code-block:: bash
+   :linenos:
+  
+  #!/bin/bash +x
+  set -e
+  set -x
+  export MODULEPATH=/soft/modulefiles/conda/:$MODULEPATH
+  module load 2023-10-04  # This is the latest conda module on Polaris
+
+  export ML_ENV=$PWD/PolarisAT/conda-envs/ml_workload_latest_conda_2 # Please change the following path accordingly 
+
+  if [[ -e $ML_ENV ]]; then
+      conda activate $ML_ENV
+  else
+      conda create  -p $ML_ENV --clone  /soft/datascience/conda/2023-10-04/mconda3/
+      conda activate $ML_ENV
+      yes | MPICC="cc -shared -target-accel=nvidia80" pip install --force-reinstall --no-cache-dir --no-binary=mpi4py mpi4py
+      yes | pip install --no-cache-dir git+https://github.com/hariharan-devarajan/dlio-profiler.git
+      pip uninstall -y torch horovod 
+      yes | pip install --no-cache-dir horovod
+      #INSTALL OTHER MISSING FILES    
+  fi
+
+Since, torchvision.datasets.ImageFolder spawns separate python processes to help the parallel data loading in torch, we will be using the `HYBRID MODE` of the DLIO Profiler, so that the application can use both APP and PRELOAD Mode to log I/O from all dynamically spawned processes and function profiling from application. 
```

Comment:
add a link to hybrid mode of the documentation 

you can add a link to the mode section of API
.. _api-modes:

and then refer to it using
:ref:`DLIO Profiler modes <api-modes>`